docs/toolchains/raspberry-pi.rst

```diff
@@ -0,0 +1,71 @@
+.. Copyright (c) 2017, Ruslan Baratov
+.. All rights reserved.
+
+Raspberry Pi
+------------
+
+.. seealso::
+
+  * `Official <https://www.raspberrypi.org/>`__
+  * `Hardware Guide <https://www.raspberrypi.org/learning/hardware-guide/>`__
+  * `Formatting SDCard <https://www.raspberrypi.org/documentation/installation/sdxc_formatting.md>`__
+  * `Download Software <https://www.raspberrypi.org/downloads/noobs/>`__
+  * `What password to use to log in after the first boot? <https://raspberrypi.stackexchange.com/q/660/70510>`__
+
+Raspbian (native)
+=================
+
+Instructions for `Raspbian <https://www.raspberrypi.org/downloads/raspbian/>`__:
+
+.. code-block:: none
+  :emphasize-lines: 3
+
+  > lsb_release -a
+  No LSB modules are available.
+  Distributor ID: Raspbian
+  Description:    Raspbian GNU/Linux 8.0 (jessie)
+  Release:        8.0
+  Codename:       jessie
+
+
+.. code-block:: none
+
+  > sudo apt-get install python3
+  > sudo apt-get install g++
+  > sudo apt-get install cmake
+
+Use ``gcc`` toolchain, e.g.:
+
+.. code-block:: none
+
+  > polly.py --toolchain gcc --verbose --config Release
+
+Cross-compiling
+===============
+
+Ubuntu
+~~~~~~
+
+.. seealso::
+
+  * `Kernel building <https://www.raspberrypi.org/documentation/linux/kernel/building.md>`__
+
+Download tools:
+
+.. code-block:: none
+
+  > git clone https://github.com/raspberrypi/tools raspberrypi-tools
+
+Save paths in environment variables:
+
+.. code-block:: none
+
+  > export RASPBERRYPI_CROSS_COMPILE_TOOLCHAIN_PATH=/.../raspberrypi-tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/bin
+  > export RASPBERRYPI_CROSS_COMPILE_TOOLCHAIN_PREFIX=arm-linux-gnueabihf
+  > export RASPBERRYPI_CROSS_COMPILE_SYSROOT=/.../raspberrypi-tools/arm-bcm2708/gcc-linaro-arm-linux-gnueabihf-raspbian-x64/arm-linux-gnueabihf/libc
```

Comment:
That's nice to know that the cross compile from `raspberrypi-tools` support the Pi 3 flags, thanks!